### PR TITLE
Add some missing assets, exception for binance ETF and test tweak

### PR DESCRIPTION
**Pull request by LefterisJP**

### CODE REVIEW

File: rotkehlchen/assets/asset.py [hunk 1]
```diff
@@ -555,6 +555,9 @@ def deserialize_from_db(cls, entry: UnderlyingTokenDBTuple) -> 'UnderlyingToken'
     strethaddress_to_identifier('0xa0246c9032bC3A600820415aE600c6388619A14D'): 'FARM',
     'STX-2': 'STX',
     strethaddress_to_identifier('0x9E46A38F5DaaBe8683E10793b06749EEF7D733d1'): 'NCT',
+    strethaddress_to_identifier('0xec67005c4E498Ec7f55E092bd1d35cbC47C91892'): 'MLN',
+    strethaddress_to_identifier('0x221657776846890989a759BA2973e427DfF5C9bB'): 'REP',
+
```
Comment (yabirgb) [inline rotkehlchen/assets/asset.py:560]: Extra white space

Comment (LefterisJP) [reply rotkehlchen/assets/asset.py:560]: argh fml

Comment (LefterisJP) [reply rotkehlchen/assets/asset.py]: fixed. Thank you for pointing it out!

**Comment by LefterisJP** [comment]
The failing e2e test passes locally with:

`ARGS="-s tests/e2e/specs/accounts-balances.spec.ts" npm run test:integration-ci -w rotki`

The failed backend one is due to remote timeout.

**Pull request merged**